This patch avoids to falsely pretend that we provide i3_with_shmlog in Slint.
diff -ru i3-4.22/meson.build i3-4.22.new/meson.build
--- i3-4.22/meson.build	2023-01-02 09:39:00.000000000 +0100
+++ i3-4.22.new/meson.build	2023-04-06 13:46:13.298295849 +0200
@@ -495,10 +495,10 @@
 )
 
 # This is the only currently working way of installing a symbolic link:
-meson.add_install_script(
-  'meson/meson-install-i3-with-shmlog',
-  get_option('bindir'),
-)
+#meson.add_install_script(
+#  'meson/meson-install-i3-with-shmlog',
+#  get_option('bindir'),
+#)
 
 executable(
   'i3bar',
